node/pkg/dal/api/controller.go

```diff
@@ -186,29 +187,36 @@ func getLatestFeeds(c *fiber.Ctx) error {
 	return c.JSON(result)
 }
 
-func getLatestFeed(c *fiber.Ctx) error {
+func getLatestMultipleFeeds(c *fiber.Ctx) error {
 	controller, ok := c.Locals("apiController").(*Controller)
 	if !ok {
 		return errors.New("api controller not found")
 	}
 
-	symbol := c.Params("symbol")
+	symbolsStr := c.Params("symbols")
 
-	if symbol == "" {
+	if symbolsStr == "" {
 		return errors.New("invalid symbol: empty symbol")
 	}
-	if !strings.Contains(symbol, "-") {
-		return errors.New("symbol should be in {BASE}-{QUOTE} format")
-	}
 
-	if !strings.Contains(symbol, "test") {
-		symbol = strings.ToUpper(symbol)
-	}
+	symbols := strings.Split(symbolsStr, ",")
+	results := make([]*dalcommon.OutgoingSubmissionData, len(symbols))
+	for i, symbol := range symbols {
+		if !strings.Contains(symbol, "-") {
+			return fmt.Errorf("wrong symbol format: %s, symbol should be in {BASE}-{QUOTE} format", symbol)
+		}
 
-	result, err := controller.Collector.GetLatestData(symbol)
-	if err != nil {
-		return err
+		if !strings.Contains(symbol, "test") {
+			symbol = strings.ToUpper(symbol)
+		}
+
+		result, err := controller.Collector.GetLatestData(symbol)
+		if err != nil {
+			return err
+		}
```

Comment:
I'm curious if we should make it fail or just skip the ones with errors, how do you think? I think this will be probably okay 🤔

Reply:
I vote for not skipping :) If we skip that might give a user an impression that they got the data they wanted. They should somehow know that they didn't get all the data they wanted. If it fails it will force them to correct their mistake and get the entirety of the data they want. But if you insist, I update 😅

Reply:
I was thinking of trailing comma which could be accidently added.. I won't insist

Reply:
sure lets keep it this way